src/requirements.txt

```diff
@@ -1,2 +1,7 @@
-python-dotenv==1.0.0
-discord.py==2.3.2
+python-dotenv
```

Comment:
always include versions in requirements. and can you briefly explain why the new packages?